__tests__/extension-requests/extension-requests.test.js

```diff
@@ -919,4 +938,31 @@ describe('Tests the Extension Requests Screen', () => {
     );
     expect(ascSortIconDisplayStyle).toBe('block');
   });
+
+  it('Should show empty message if all extension requests have been addressed', async () => {
```

Comment:
Is there a check, if the API returns no data?

Reply:
This test works even if no data is returned from the API. Is an explicit check still required?

Reply:
but this test doesn't check for the empty case